.eggs/README.txt

```diff
@@ -0,0 +1,9 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
Please do not include the `.eggs` directory in the repository.

Reply:
Noted.